update.sh

```diff
@@ -3,9 +3,9 @@
 ############## Begin Function Section ##############
 
 check_online_status() {
-  CHECK_ONLINE_IPS=(1.1.1.1 9.9.9.9 8.8.8.8)
-  for ip in "${CHECK_ONLINE_IPS[@]}"; do
-    if timeout 3 ping -c 1 ${ip} > /dev/null; then
+  CHECK_ONLINE_DOMAINS=('https://github.com')
```

Comment:
Maybe add one or two domains here as well.

google.com and hub.docker.com i would say.

The rest looks good

Reply:
Had added only the used domains by intention.

Reply:
i would at least add hub.docker.com as we pulling images from there.